File: Python_Programming/MongoDB_229702/insert.py
from logging import exception
from pymongo import MongoClient
client = MongoClient("mongodb://localhost:27017/")
db = client.PradMCC
def insert():
    try:
        name1 = input("Enter the name: ")
        age1 = int(input("Enter the age: "))
        db.myMcc.insert_one({
            "Name": name1,
            "Age": age1
        })
        print("Data inserted successfully")
    except Exception:
        print(str(e))
insert()

File: Java_Programming/java_with_MongoDB/insert.java
import com.mongodb.client.MongoCollection;
import com.mongodb.client.MongoDatabase;
import com.mongodb.MongoClient;
import com.mongodb.MongoCredential;
import org.bson.Document;

public class insert {
    public static void main(String args[]) {
        MongoClient mongo = new MongoClient("localhost", 27017);
        System.out.println("Connected to DataBase Sucessfully ");
        MongoDatabase database = mongo.getDatabase("TYITDB1");

        MongoCollection<Document> Collection = database.getCollection("TYITCOL");
        System.out.println("Collection Sample Collection Selected Successfully ");

        Document document = new Document();
        document.append("ID", 3);
        document.append("Description", "database 2");
        document.append("Likes", 300);
        document.append("URL", "https://www.mongoDb.com");
        document.append("by", "NOSQL");
        Collection.insertOne(document);
        System.out.println("Document inserted Successfully");
    }
}
